standard.go

```diff
@@ -0,0 +1,148 @@
+package pagerduty
+
+import (
+	"context"
+
+	"github.com/google/go-querystring/query"
+)
+
+const (
+	standardPath = "/standards"
+)
+
+// Standard defines a PagerDuty's resource standard.
+type Standard struct {
+	Active       bool                         `json:"active"`
+	Description  string                       `json:"description,omitempty"`
+	Exclusions   []StandardInclusionExclusion `json:"exclusions,omitempty"`
+	ID           string                       `json:"id,omitempty"`
+	Inclusions   []StandardInclusionExclusion `json:"inclusions,omitempty"`
+	Name         string                       `json:"name,omitempty"`
+	ResourceType string                       `json:"resource_type,omitempty"`
+	Type         string                       `json:"type,omitempty"`
+}
+
+type StandardInclusionExclusion struct {
+	Type string `json:"type,omitempty"`
+	ID   string `json:"id,omitempty"`
+}
+
+// ListStandardsResponse is the data structure returned from calling the ListStandards API endpoint.
+type ListStandardsResponse struct {
+	Standards []Standard `json:"standards"`
+}
+
+// ListStandardsOptions is the data structure used when calling the ListStandards API endpoint.
+type ListStandardsOptions struct {
+	Active bool `url:"active,omitempty"`
+
+	// ResourceType query for a specific resource type.
+	//  Allowed value: technical_service
+	ResourceType string `url:"resource_type,omitempty"`
+}
+
+type ResourceStandardScore struct {
+	ResourceID   string             `json:"resource_id,omitempty"`
+	ResourceType string             `json:"resource_type,omitempty"`
+	Score        *ResourceScore     `json:"score,omitempty"`
+	Standards    []ResourceStandard `json:"standards,omitempty"`
+}
+
+type ResourceScore struct {
+	Passing int `json:"passing,omitempty"`
+	Total   int `json:"total,omitempty"`
+}
+
+type ResourceStandard struct {
+	Active      bool   `json:"active"`
+	Description string `json:"description,omitempty"`
+	ID          string `json:"id,omitempty"`
+	Name        string `json:"name,omitemtpy"`
+	Pass        bool   `json:"pass"`
+	Type        string `json:"type,omitemtpy"`
+}
+
+type ListMultiResourcesStandardScoresResponse struct {
+	Resources []ResourceStandardScore `json:"resources,omitempty"`
+}
+
+type ListMultiResourcesStandardScoresOptions struct {
+	// Ids of resources to apply the standards. Maximum of 100 items
+	IDs []string `url:"ids,omitempty,brackets"`
+}
+
+// ListStandardsWithContext lists all the existing standards.
+func (c *Client) ListStandardsWithContext(ctx context.Context, o ListStandardsOptions) (*ListStandardsResponse, error) {
```

Comment:
nit: Some of these methods are `*WithContext` while others are not. Maybe we just drop the `*WithContext` since not including the context on API functions is now deprecated?

Reply:
Yes, You are right! There shouldn't be more `*WithContext` for new methods implementations. Thank you for the reminder ✌🏽